lib/chef/knife/cloud/vcenter_service.rb

```diff
@@ -97,10 +97,14 @@ def create_server(options={})
           case options[:type]
           when "clone"
 
+            datacenter_exists?(options[:datacenter])
+
             # Some of ht eoptions need to be the ID of the component in VMWAre
             # Update these using the REST API so that they can be passed to the support library
             options[:targethost] = get_host(options[:targethost])
 
+            options[:resource_pool] = get_resource_pool(options[:resource_pool])
```

Comment:
We should add this to be an option should be able to declare. Line 136, and have a note that it defaults to the first one pool if not declared. We need a way to set to the "root" data center too? maybe `--folder /`?